### ISSUE TITLE
Allow apocrypha in MyBible SQLite3 modules parsed and displayed in the app

### ISSUE THREAD

**Issue by viktor-zhuromskyy** [AndBible/and-bible#2491]
My modules have several apocrypha books such and Book of Enoch and Didache, as well as deuterocanonical and extra Septuagint books like Susanna and Bell and Dragon, that are Daniel 13 & 14,.. but I noticed the AndBible does not parse them from MyBible .SQLite3 modules.

Would be so happy if you take y request as a feature request.

**Comment by tuomas2** [comment]
Do MyBible modules announce their versification system somehow that we could use it instead of our default guess which is KJV (if I remember correctly)? I think apocrypha books are displayed if a "sword-like" module is made and versification system is configured properly. See https://github.com/AndBible/and-bible/wiki/MyBible-module-support#module-creators

**Comment by tuomas2** [comment]
No, default versification is actually KJVA, so apocrypha should be supported. Maybe there is a bug then.

**Comment by tuomas2** [comment]
to help fixing, provide exact module that the issue is reproducable.

**Comment by viktor-zhuromskyy** [comment]
> Do MyBible modules announce their versification system somehow that we could use it instead of our default guess which is KJV (if I remember correctly)? I think apocrypha books are displayed if a "sword-like" module is made and versification system is configured properly. See https://github.com/AndBible/and-bible/wiki/MyBible-module-support#module-creators

MyBible pulls all the module data from SQLite3 database in info table and books table.

**Comment by viktor-zhuromskyy** [comment]
For example this MB module https://s2.igrnt.info/MyBible/.mb_private_repo/full_zip/IGRB%2720r%2B.zip and https://s2.igrnt.info/MyBible/.mb_private_repo/full_zip/LCV%2720r%2B.zip does have book of Enoch and Didache.

**Comment by tuomas2** [comment]
811 is enoch, and 511 is didache I assume. 1. Enoch is supported by JSword (but it is not part of KJVA versification, so it is not displayed there.. but it is displayed if versification is changed to LXX), but I can't find Didache in any bible book listings.

Added support for 811 book code to MyBibleBookMap.kt aab3a30be88d795234c410c0fda58561215a5f9f

**Comment by tuomas2** [comment]
If your module is following LXX versification then you could use LXX in your module config ( as instructed [here](https://github.com/AndBible/and-bible/wiki/MyBible-module-support#module-creators) )

**Comment by viktor-zhuromskyy** [comment]
> 811 is enoch, and 511 is didache I assume. 1. Enoch is supported by JSword (but it is not part of KJVA versification, so it is not displayed there.. but it is displayed if versification is changed to LXX), but I can't find Didache in any bible book listings.
> 
> Added support for 811 book code to MyBibleBookMap.kt [aab3a30](https://github.com/AndBible/and-bible/commit/aab3a30be88d795234c410c0fda58561215a5f9f)

Thanks a lot. Can you please add 511 book number as Didache? I am the only one publishing Didache in MyBible and because of missing availability of Didache allocation in MyBible I used 511 since it is not occupied.

**Comment by viktor-zhuromskyy** [comment]
> If your module is following LXX versification scheme then you could use LXX in your module config ( as instructed [here](https://github.com/AndBible/and-bible/wiki/MyBible-module-support#module-creators) )

If I use LXX versification, will it pull up New Testament books as well?

Is there any way to create allocation file with books versification locally in my module configuration?

**Comment by viktor-zhuromskyy** [comment]
> > 811 is enoch, and 511 is didache I assume. 1. Enoch is supported by JSword (but it is not part of KJVA versification, so it is not displayed there.. but it is displayed if versification is changed to LXX), but I can't find Didache in any bible book listings.
> > Added support for 811 book code to MyBibleBookMap.kt [aab3a30](https://github.com/AndBible/and-bible/commit/aab3a30be88d795234c410c0fda58561215a5f9f)
> 
> Thanks a lot. Can you please add 511 book number as Didache? I am the only one publishing Didache in MyBible and because of missing availability of Didache allocation in MyBible I used 511 since it is not occupied.

Can you please point me to the files in the repo tree that declare versification schemes?

**Comment by tuomas2** [comment]
They are defined in JSword, i.e. here:

https://github.com/AndBible/jsword/tree/master/src/main/java/org/crosswire/jsword/versification

Please make PR towards our fork of AndBible (for the time being - there's process together with Step Bible guys to start maintaining official repository but it's not there yet)..
